src/groupByKey.ts

```diff
@@ -2,6 +2,22 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(items: [], key: string): GroupsMap<string> {
+  const itemsValuesOfKeys: string[] = [];
```

Comment:
try to simplify this logic without using additional arrays, you can just check `if (!results[item]) {`